src/pcrlock/pcrlock.c

```diff
@@ -1514,16 +1513,20 @@ static EventLogComponent *event_log_find_component(EventLog *el, const char *id)
         EventLogComponent k = {
                 .id = (char*) id,
         };
-        EventLogComponent *kk = &k;
+        EventLogComponent *kk = &k, **found;
```

Comment:
these changes here and below do not look related to the 'pcrlock: add pre-defined pcrlock files' commit they are in? rebase gone wrong?